controllers/storagecluster/storageconsumer.go

```diff
@@ -0,0 +1,55 @@
+package storagecluster
+
+import (
+	"fmt"
+
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v4/v1"
+	ocsv1a1 "github.com/red-hat-storage/ocs-operator/api/v4/v1alpha1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const (
+	localStorageConsumerName          = "internal"
+	localStorageConsumerConfigMapName = "internal-resources"
```

Comment:
```suggestion
	localStorageConsumerConfigMapName = "storage-consumer-internal"
```

Reply:
@leelavg 
I think we now need to change this to `storageconsumer-internal`

Reply:
as discussed, changed to `storageconsumer-internal`.